src/core/injections/controller/controller.py

```diff
@@ -360,13 +357,12 @@ def injection_proccess(url, check_parameter, http_request_method, filename, time
     if not header_name == settings.COOKIE and not the_type == "HTTP Header":
       settings.CHECKING_PARAMETER = checks.check_http_method(url)
       settings.CHECKING_PARAMETER += ('', ' JSON')[settings.IS_JSON] + ('', ' SOAP/XML')[settings.IS_XML]
-    if header_name == settings.COOKIE :
-       settings.CHECKING_PARAMETER += str(header_name) + str(the_type) + str(inject_parameter)
-    else:
-       settings.CHECKING_PARAMETER += str(the_type) + str(header_name) + str(inject_parameter)
+    settings.CHECKING_PARAMETER = f"{the_type}{header_name}{inject_parameter}"
+    if header_name == settings.COOKIE:
+        settings.CHECKING_PARAMETER = f"{header_name} {the_type}{inject_parameter}"
 
     if check_parameter in settings.CUSTOM_INJECTION_MARKER_PARAMETERS_LIST:
-      settings.CHECKING_PARAMETER = "(custom) " + settings.CHECKING_PARAMETER
+      settings.CHECKING_PARAMETER = f"(custom) {settings.CHECKING_PARAMETER}"
```

Comment:
For Python `2.x` and `3.x` compatibility, avoid `f-strings` (they’re Python 3.6+ only)